### DISCUSSION TITLE
Summary of changes to the version 1.1.0

### DISCUSSION THREAD

**Discussion by dmitry-kazakov** [dokan-dev/dokany#1028] [Q&A] [answered]
As I maintainer of Ada bindings to dokan, I am interested in the summary of changes in the API made since the version 1.1.0.

**Comment by Liryna** [comment, marked as answer]
Hi @dmitry-kazakov ,

You can take a look at the `dokan.h` diff between 1.1.0 and the latest 1.5:
https://github.com/dokan-dev/dokany/compare/v1.1.0...v1.5.0.3000#diff-043a92293d2218cfe605149daa53485589f1f260f6224f7d858ed20e6c5b3e61

The API hasn't moved much except the added `DokanNotify*` and `DokanRemoveMountPointEx` being removed.

Is there any issue you are facing ?

Thanks for creating the Ada bindings! let me know when you will want me to add it to https://github.com/dokan-dev/dokany/wiki/Dokan-API-Wrappers-language-compatible

**Comment by dmitry-kazakov** [comment]
Thank you for the response. I asked because there was a lot of important changes from 0.6.0 to 1.1.0.

I downloaded 1.5.0 3000 and so far saw no issues at all. The Ada implementation of the memory-mapped file system worked without modifications out of the box.

If you decide to add a link to Ada bindings, here is the page:

www.dmitry-kazakov.de/ada/dokan.htm

**Reply by Liryna** [reply]
Done! https://github.com/dokan-dev/dokany/wiki/Dokan-API-Wrappers-language-compatible Thanks again 🚀
